client/src/components/existing-request/existing-request-display.vue

```diff
@@ -310,15 +310,20 @@ export default class ExistingRequestDisplay extends Mixins(
 
   private get reviewDate () {
     if (this.nr.waiting_time) {
-      let waitingTime = this.nr.waiting_time
+      let queueTime = this.nr.waiting_time
+      let submittedDate = Moment(this.nr.submittedDate)
+      let todaysDate = Moment()
+      // get number of days since the nr was submitted
+      let diffDays = todaysDate.diff(submittedDate, 'days')
```

Comment:
As Moment is large (and we don't REALLY need it nowadays), I want to get away from Moment actually. (And I've updated most of the code in NR Release 2.0 branch.)

See if src/components/mixins/date-mixin.ts::daysFromToday() will work for you.

![image](https://user-images.githubusercontent.com/10002889/111386025-dc8ed100-8668-11eb-8a06-b68d1b213d73.png)

Reply:
One problem with dates/times -- that Moment doesn't solve but my new code does -- is that the browser's date/time could be spoofed to try to get different UI/API behaviour. "Today" and "now" needs to be in Pacific time regardless of what the browser thinks. Moment solves the tz issue but doesn't solve this issue because it assumes the browser's UTC time is correct.

Some of my new code, if you want to have a look at how to do it with plain old TS/JS: https://github.com/bcgov/business-edit-ui/blob/master/src/mixins/date-mixin.ts

Reply:
I changed the code to use daysFromToday